include/odp/api/spec/proto_stats.h

```diff
@@ -104,6 +128,10 @@ odp_proto_stats_t odp_proto_stats_lookup(const char *name);
  *
  * Destroy a proto stats object already created.
  *
+ * - Before destroying proto stats object having ODP_PROTO_STATS_ID_TX_*
+ *   enabled, for all PKTIO devices to which packets were Tx'ed earlier with
+ *   this proto stats object, odp_pktio_stop() must be called.
```

Comment:
This sounds ok, but maybe this should specify an additional restriction that existing packets that refer to the proto stats object being destroyed, must not be sent.

Reply:
I meant, ...must not be sent at the same time as or after the proto stats object destruction.